src/components/structures/TimelinePanel.tsx

```diff
@@ -1602,7 +1613,7 @@ class TimelinePanel extends React.Component<IProps, IState> {
             if (!node) continue;
 
             const boundingRect = node.getBoundingClientRect();
-            if (boundingRect.top <= wrapperRect.bottom && boundingRect.bottom >= wrapperRect.top) {
+            if (boundingRect.top <= screenBottom && boundingRect.bottom >= screenTop) {
                 enteredVisibleRange = true;
```

Comment:
I'm still worried about this causing flickering:
 * there is an undecryptable event right at the top of the window
 * so we display the error bar
 * which pushes the undecryptable event outside the visible range
 * so we hide the error bar
 * repeat

Am I missing something? Or do we need some hysteresis here?

Reply:
Good point, added hysteresis.